django-pagination/terms/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path(
+        "all", views.AllKeywordsView.as_view(template_name="terms/base.html")
```

Comment:
:warning: **Fact Check**

What about the other views you mentioned in the tutorial? It would be helpful and convenient for the readers to include them here as well.

Reply:
Hmm … this makes me think if I approached the materials repo the wrong way. 🤔 

My idea was to have the project as a starting point. However, maybe it’d be even better to provide the final project with all the examples I show in the tutorial included. What do you think?